server/templates/student/assignment/code.html

```diff
@@ -66,6 +66,9 @@ <h3 class="text-center">
             {% for filename, file in files | dictsort %}
               {{ diff.source_file(backup, filename, file) }}
             {% endfor %}
+            {% for filename, file in external_files | dictsort %}
+              {{ diff.external_file(backup, filename, file) }}
+            {% endfor %}
```

Comment:
I'd like these to all be in alphabetical order somehow. Since this code is also in the staff template perhaps we can macrotize it